docker_app/main.py

```diff
@@ -0,0 +1,162 @@
+import re
+import datetime
+import typing
+
+import docker  # type: ignore
+from docker.models import images, containers  # type: ignore
+
+
+class DockerModule:
+    def __init__(self):
+        """
+        Connect to local docker and create docker-client connection
+        """
+        self.client = docker.from_env()
+
+    def close(self) -> None:
+        """
+        Close docker-client connection
+        """
+        self.client.close()
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        self.close()
+        if exc_value:
+            raise
+
+    """
+    Images
+    """
+
+    def get_images(self) -> typing.List[images.Image]:
+        """
+        Return the list of available image objects
+
+        :return: List of image objects
+        """
+        all_images = self.client.images.list()
+
+        return all_images
+
+    def build_new_image(
+        self, site_name: str, site_dir: str, tag: str = ""
+    ) -> typing.Tuple[images.Image, typing.Iterable]:
+        """
+        Build new image from src
+
+        :param site_name: Site name which will be used in image name
+        :param site_dir: Path to Lektor folder with site
+        :param tag: Additional param for default tag
+
+        :return: Tuple with image object and logs.
+        """
+
+        datetime_now = datetime.datetime.now()
+
+        # clean site name and tag from invalid symbols
+        clean_site_name = re.sub(
+            r'[ !"#$%&()*+,./:;<=>?@[\]^`{|}~]', "_", site_name
+        ).lower()
+
+        if tag:
+            # use sended person tag, but sirstly prepare and clean it
```

Comment:
```suggestion
            # use the sent person tag, but first prepare and clean it
```

I also don't quite get what "person" are we talking about